src/layers/base-layer.ts

```diff
@@ -1064,13 +1067,13 @@ class Layer {
     const dataUpdateTriggers = this.getDataUpdateTriggers(layerDataset);
     const triggerChanged = this.getChangedTriggers(dataUpdateTriggers);
 
-    if (triggerChanged.getMeta) {
+    if (triggerChanged && triggerChanged.getMeta) {
```

Comment:
`triggerChanged?.getMeta`

Reply:
Type of triggerChanged is:
`const triggerChanged: false | {
    [key: string]: true;
}`